Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3-onbuild
+FROM python:3.6-stretch
```

Comment:
I assume if you use`stretch` you'll need to install `libsqlite3-mod-spatialite`

Reply:
I am not sure about it, but it was failing when we use onbuild so i used stretch.

![Screenshot from 2019-06-03 11-06-58](https://user-images.githubusercontent.com/20985005/58778362-de041c00-85ef-11e9-8584-fc321ab765cb.png)